src/pages/verify/getting-started/oidc-intro.mdx

```diff
@@ -344,6 +348,95 @@ The token will be issued upon successful user identification:
 }
 ```
 
+## Authenticate with Headless
+
+Headless flow is a custom protocol provided by Criipto that is very similar to <a href="https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html" target="_blank">CIBA</a> but with a few key differences.
+
+Where CIBA requires that the user is known in advance so that a request may be sent directly to the authentication device, headless instead returns values to the relying part so that they may trigger necessary UI or interaction elements with the user directly.
+
+Headless is an extension to OpenID Connect. However, unlike in other OpenID Connect flows, there is a direct communication between your application and the OpenID Provider (Criipto Verify), without redirects through the user's browser.
+
+Headless supports use cases not covered by other OpenID Connect flows, such as: 
+
+- Rendering UI elements in your own webpage instead of via a Criipto redirect
+- Performing app2app where a browser is not required
+
+Criipto Verify supports Headless for the Swedish BankID
+
+### Example request
+
+The client application shall make an `HTTP POST` request to the backchannel authentication endpoint to ask for end-user authentication. 
```

Comment:
I think this reads better:
```suggestion
The client application sends an `HTTP POST` request to the backchannel authentication endpoint to ask for end-user authentication. 
```

Reply:
Copy-pasta from the CIBA docs so no real feelings about it for me